packages/FileUpload/src/index.tsx

```diff
@@ -168,18 +168,20 @@ const FileUpload: React.SFC<IFileUploadProps> = ({
 
   const handleInputChange = ({ event }: any) => {
     const {
-      target: { id, name, value }
+      target: { id: _id, name, value }
     } = event;
 
     const filesToUploadCopy = cloneDeep(filesToUpload);
+
     const previewImagesCopy = cloneDeep(previewImages);
+    const position = _id.split("-")[0];
```

Comment:
maybe a better name is `index` but it's more of a personal preference

Reply:
Yes, it's probably clearer so I just changed it